packages/flame_rive/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+## [next]
+
+## 1.0.0-releasecandidate.1
```

Comment:
This file is not inline without other changelogs: https://raw.githubusercontent.com/flame-engine/flame/main/packages/flame/CHANGELOG.md